src/transformers/onnx/features.py

```diff
@@ -303,8 +303,16 @@ def get_model_from_feature(feature: str, model: str) -> Union[PreTrainedModel, T
             The instance of the model.
 
         """
+        # If PyTorch and TensorFlow are installed in the same environment, we
+        # load an AutoModel class by default
         model_class = FeaturesManager.get_model_class_for_feature(feature)
-        return model_class.from_pretrained(model)
+        try:
+            model = model_class.from_pretrained(model)
+        # Load TensorFlow weights in an AutoModel instance if PyTorch and
+        # TensorFlow are installed in the same environment
+        except OSError:
```

Comment:
Are you sure it's the only possible exception that can be raised?
It seems like `EnvironmentError` and `ValueError` can be raised too according to [here](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_utils.py#L1315) and [here](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_utils.py#L1335).